stdlib/public/core/FlatMap.swift

```diff
@@ -44,10 +49,13 @@ extension LazySequenceProtocol {
 }
 
 extension LazyCollectionProtocol {
-  /// Returns the concatenated results of mapping `transform` over
-  /// `self`.  Equivalent to 
+  /// Returns the concatenated results of mapping the given transformation over
+  /// this collection.
```

Comment:
I'm only using "concatenated" when the result concatenates sequences/collections, not the optional unwrapping `flatMap`